docs/how-to/install-slice.rst

```diff
@@ -0,0 +1,137 @@
+How to install your own package slice ?
+***************************************
+
+When a specific package slice is not available on the `upstream Chisel releases <https://github.com/canonical/chisel-releases>`_, you will more likely end up creating your own slice definition.
+
+Once you have it though, the most obvious question is: **how can I install this custom slice with Chisel?**
+
+Let's assume you want to install the OpenSSL binaries slice created in :ref:`here <create-slice>`...
+
+**First**, clone the Chisel releases repository:
+
+.. code-block:: sh
+
+    # Let's assume we are working with Ubuntu 22.04
+    $ git clone -b ubuntu-22.04 https://github.com/canonical/chisel-releases/
+
+This repository acts as the database of slice definitions files for each Chisel release (Chisel releases are named analogously to Ubuntu releases, and mapped into Git branches within the repository).
+
+Chisel will only recognize slices belonging to a Chisel release, so you need to copy your slice definitions file - ``openssl.yaml`` in this example - into the ``chisel-releases/slices`` folder. Note that if a slice definitions file with the same name already exists, it most likely means that the package you're slicing has already been sliced before, and in this case, you only need to merge your changes into that existing file.
+
+At this point, you should be able to find your custom OpenSSL slice ``bins`` in the local Chisel release:
+
+.. code-block:: sh
+
+    grep -q "bins" chisel-releases/slices/openssl.yaml && echo "My slice exists"
+
+If you wanted to test it with Chisel alone, you could now simply run ``chisel cut --release ./chisel-releases --root . openssl_bins``. You should end up with a few folders, amongst which there would be ``./usr/bin/openssl``.
```

Comment:
might be a good idea to use a different `--root`, to avoid polluting the current working directory